Alicerce/Sources/Logging/Log+NodeLogDestination.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  Log+NodeLogDestination.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public extension Log {
+
+    public class NodeLogDestination : LogDestination, LogDestinationFallible {
```

Comment:
As discussed, I think this Destination should be converted into a more reusable one, and should not be tied to a single request type or a type of server (Node).